accepted/PSR-12-extended-coding-style-guide.md

```diff
@@ -882,10 +888,19 @@ All operators not described here are left undefined.
 
 ### 6.1. Unary operators
 
-The increment/decrement operators MUST NOT have any space between
-the operator and operand.
+The increment/decrement operators, the logical not operator, the conversion 
+and negation operators MUST NOT have any space between the operator and operand.
```

Comment:
Those are new operators added. It's not a simple fix.

Reply:
Agree, this is NOT a simple cosmetic/semantic rework, this is an extension of what was herebefore written. Do you think it damages the spec?

Reply:
Without this, I could merge this mysels as a secretary, as simple typos/fixes. Since this is a change, the editor (@dragonmantank) has the final say on that.